src/diffpy/labpdfproc/tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def user_filesystem(tmp_path):
+    directory = Path(tmp_path)
+    os.chdir(directory)
+
+    existing_dir = Path(tmp_path).resolve() / "existing_dir"
```

Comment:
since you are making the full stack of files on an `input_directory` we don't need the `existing_dir` and `existing_file` any more.